File: app/src/main/java/xyz/iridiumion/enlightened/util/RandomUtils.java
package xyz.iridiumion.enlightened.util;

/**
 * Author: 0xFireball
 */
public class RandomUtils {
    public static String getFileExtension(String fileName) {
        int dotIndex = fileName.lastIndexOf(".");
        if (dotIndex == -1)
            return "";
        return fileName.substring(dotIndex + 1, fileName.length());
    }
}

File: app/src/main/java/xyz/iridiumion/enlightened/util/RandomUtils.kt
package xyz.iridiumion.enlightened.util

/**
 * Author: 0xFireball
 */
object RandomUtils {
    fun getFileExtension(fileName: String): String {
        val dotIndex = fileName.lastIndexOf(".")
        if (dotIndex == -1)
            return ""
        return fileName.substring(dotIndex + 1, fileName.length)
    }
}
